modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/model/SocialTrafficChannel.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.analytics.reports.web.internal.model;
+
+import com.liferay.analytics.reports.web.internal.model.util.TrafficChannelUtil;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.util.ResourceBundleUtil;
+
+import java.util.Locale;
+import java.util.Objects;
+import java.util.ResourceBundle;
+
+/**
+ * @author David Arques
+ */
+public class SocialTrafficChannel implements TrafficChannel {
```

Comment:
Don't include Social in this ticket, this should be moved to https://issues.liferay.com/browse/LPS-119486

Reply:
Well, I'm adding the first level of social, which already existed before the refactor. Do you mind if I leave it as it is and add the details in the corresponding ticket (https://issues.liferay.com/browse/LPS-119486)?